<launch>

  <arg name="server" default="optitrack"/>
  <arg name="namespace" default="arducopter"/>

  <node pkg="vrpn_client_ros" type="vrpn_client_node" name="vrpn_client_node" output="screen">
    <rosparam subst_value="true">
      server: $(arg server)
      port: 3883

      update_frequency: 100.0
      frame_id: world

      # Use the VRPN server's time, or the client's ROS time.
      use_server_time: false
      broadcast_tf: false

      # Must either specify refresh frequency > 0.0, or a list of trackers to create
      refresh_tracker_frequency: 1.0
      #trackers:
      #- FirstTracker
      #- SecondTracker
    </rosparam>
  </node>

  <group ns="$(arg namespace)">
    <node pkg="mmuav_arducopter_bridge" type="vrpn_transformer.py" name="vrpn_transformer" output="screen">
      <param name="topic_name" value="/vrpn_client_node/arducopter/pose" />
      <param name="median_filter_length" value="5" />
    </node>
  </group>

</launch>